src/offlinelogstorage/dlt_offline_logstorage_behavior.c

```diff
@@ -370,15 +370,9 @@ int dlt_logstorage_storage_dir_info(DltLogStorageUserConfig *file_config,
         config->records = NULL;
     }
 
-    char* suffix = NULL;
-    for (i = 0; i < cnt; i++) {
-        if (config->gzip_compression) {
-            suffix = strdup(".dlt.gz");
-        }
-        else {
-            suffix = strdup(".dlt");
```

Comment:
Hello @averater,
strdup will allocate new memory, so this variable is freed after using